File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.java
class Solution {
    public String removeOuterParentheses(String s) {
        String result = "";
        int count = 0;
        for(char c : s.toCharArray()){
            if(c == '('){
                if(count != 0){
                    result += c;
                }
                count++;
            }
            else{
                count--;
                if(count != 0){
                    result += c;
                }
            }
        }
        return result;
    }
}

File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.py
class Solution(object):
    def removeOuterParentheses(self, s):
        count = 0
        result = ""
        for char in s:
            if(char == '('):
                if(count != 0):
                    result += char
                count += 1
            else:
                count -= 1
                if(count != 0):
                    result += char
        return result

                
        